minard/static/js/scatter_plot_rates.js

```diff
@@ -0,0 +1,119 @@
+function history() {
+    var params = {}
+    params['crate'] = document.getElementById("crate-sel").value;
+    params['slot'] = document.getElementById("slot-sel").value;
+    params['channel'] = document.getElementById("channel-sel").value;
+    window.location.replace($SCRIPT_ROOT + "/check_rates_history?" + $.param(params));
+}
+
+var data = window.data;
+if( data != undefined && data != "" ){
+    var data_length = data.length;
+    var last_run = data[0][0];
+    var first_run = data[data_length-1][0]
+    var len = Math.floor((last_run - first_run)/data.length);
+    var run_offset = 0;
+    var start_offset = 103214;
+    var start = start_offset - run_offset;
+    var end = d3.max(data, function(d) { return d[0]; }) + 1 - run_offset; 
+
+    var margin = {top: 10, right: 80, bottom: 80, left: 120}
+        ,width = $("#main").width()
```

Comment:
I think you want to subtract the margins from this so it fits on one page (https://github.com/snoplus/minard/blob/master/minard/static/js/histogram.js#L25).